core/variant_call.cpp

```diff
@@ -1438,13 +1438,12 @@ Variant::Type Variant::get_method_return_type(Variant::Type p_type, const String
 }
 
 Vector<Variant> Variant::get_method_default_arguments(Variant::Type p_type, const StringName &p_method) {
+	ERR_FAIL_INDEX_V(p_type, Variant::VARIANT_MAX, Vector<Variant>());
 	const _VariantCall::TypeFunc &tf = _VariantCall::type_funcs[p_type];
 
 	const Map<StringName, _VariantCall::FuncData>::Element *E = tf.functions.find(p_method);
-	if (!E) {
-		return Vector<Variant>();
-	}
-
+	ERR_FAIL_COND_V(!E, Vector<Variant>());
+	
```

Comment:
There seems to be trailing whitespace on this line, according to style checks.

Reply:
fixed